packages/mui-lab/src/index.d.ts

```diff
@@ -22,21 +22,12 @@ export * from './ClockPicker';
 export { default as DatePicker } from './DatePicker';
 export * from './DatePicker';
 
-export { default as DateRangePicker } from './DateRangePicker';
```

Comment:
For these components, should we instead of just removing them, create an empty components in the lab that will throw an error and return null and maybe the error can point to the blogpost we have? Something like https://github.com/mui/material-ui/blob/master/packages/mui-lab/src/Alert/Alert.js but we won't return the real component, we would return `null`.

Reply:
Good idea, I will add it

Reply:
Done, I tried to handle the types and the utility methods up to a certain point